src/pages/tests/button.hbs

```diff
@@ -5,7 +5,7 @@ layout: blank
 <h1 class="drizzle-b-h1">
   Buttons
 </h1>
-<ul class="sprk-b-List--bare">
+<ul class="sprk-b-List--bare sprk-u-mbh">
```

Comment:
Added a test case to the buttons test page for centered buttons at larger screens and full size buttons at smaller screens.